api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -267,6 +327,12 @@ public List<GlobalProperty> saveGlobalProperties(List<GlobalProperty> props) thr
 	public GlobalProperty saveGlobalProperty(GlobalProperty gp) throws APIException {
 		// only try to save it if the global property has a key
 		if (gp.getProperty() != null && gp.getProperty().length() > 0) {
+			
```

Comment:
We should still throw an exception even if the global property has a null value. Because this would still be an unauthorised access attempt.

Reply:
@dkayiwa won't this affect the global property saving process?!

Reply:
Do you mean that originally, when one who does not have the required privilege, tried to save a global property with a null value, we just returned null?

Reply:
no, sorry, that was a mistake😊